### ISSUE TITLE
The function `simple` is missing from type definitions

### ISSUE THREAD

**Issue by illright** [morganney/swc-walk#9]
Hi! Glad you made this project, I've been looking for something to walk an SWC tree myself and was frustrated not to find anything.

When I install this package and try to import `{ simple }` from it, VS Code doesn't suggest it. Clicking into the dependency revealed that the main `.d.ts` file only exports types and not functions:

```ts
// node_modules/.pnpm/swc-walk@1.0.0-rc.1/node_modules/swc-walk/dist/esm/types.d.ts

// ... a bunch of exported types ...

export {};
```

Am _I_ missing something, or are the type defintions?

**Comment by illright** [comment]
For now I worked around it by doing this in my code:

```ts
import { simple } from 'swc-walk'

declare module 'swc-walk' {
  export const simple: Simple
}
```

**Comment by morganney** [comment]
Thanks for reporting this. The project is still a work in progress. This should be fixed in v1.0.0-rc.2.
